music-journal/pages/api/list-memories.js

```diff
@@ -26,13 +26,9 @@ async function handler(req, res) {
             });
         }
 
-        let memories = []
-
         const memoriesRef = await database.ref(`/memory/${user.uid}`);
-
-        await memoriesRef.orderByValue().once("value", snapshot => {
-            snapshot.forEach(data => { memories.push(data.val()) })
-        })
+        const memoriesSnapshot = await memoriesRef.orderByValue().once("value")
+        const memories = Object.assign({}, memoriesSnapshot.val())
```

Comment:
ah yeah this is what you linked me!

Reply:
Yeah I much prefer that style to the callbacks in callbacks